CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file. If a contri
 
 ## Unreleased
 
+- Fix off-by-one error in insertRuleHelpers.js, my @migueloller (see #1749)
```

Comment:
Can you add the two links for the PR and your username please? There’s also a small typo: “my”

Reply:
Whoops!